packages/@react-spectrum/card/test/CardView.test.js

```diff
@@ -1164,4 +1164,33 @@ describe('CardView', function () {
       expect(row.parentNode.style.height).toBe(`${mockHeight}px`);
     });
   });
+
+  // TODO: not testing waterfall layout because of aforementioned issue with the heights for each card being set to 0 for that layout
+  it.each`
+    Name                  | layout
+    ${'Grid layout'}      | ${GridLayout}
+    ${'Gallery layout'}   | ${GalleryLayout}
+  `('$Name CardView should only scroll an item into view when in keyboard modality', function ({layout}) {
```

Comment:
Second test to confirm no scroll with pointer event?

Reply:
good idea, will add

Reply:
Ah actually I do that in this test: https://github.com/adobe/react-spectrum/pull/2725/files/7b8c56678028193bb1bcf6e621aade9851289351#diff-3ef103560236269b49ca02cc46a7b9f7894b45739ce00c38da98ab4b385abe65R1181-R1186